inc/saiport.h

```diff
@@ -2104,6 +2104,21 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_SUPPORTED_LINK_TRAINING_MODE,
 
+    /**
+     * @brief Fabric port isolation setting.
+     *
+     * true: The link may be enabled in serdes level and the
+     * MAC level, but the link partner will not use
+     * it for traffic distribution.
+     * false: Undo the isolation operation.
+     * This attribute is for fabric links only.
```

Comment:
can we add something like this in the comment section. 
Refer sairouterinterface.h

@condition  SAI_PORT_ATTR_TYPE == SAI_PORT_TYPE_FABRIC

Reply:
Hi, 

SAI_PORT_ATTR_TYPE is a read only attribute , and as far as I know that @condition not works for read only attribute.  So checking with you again about if we need to make this change ? 

thank you 

Jie

Reply:
I tried add the @condition, and the metadata check failed, 

it looks like based on the following 3 checks, we can not use @condition check for read only attribute. thank you 

https://github.com/opencomputeproject/SAI/blob/f69394f590362b1b02852ec80f00f9a27a02ed97/meta/saisanitycheck.c#L1284
https://github.com/opencomputeproject/SAI/blob/f69394f590362b1b02852ec80f00f9a27a02ed97/meta/saisanitycheck.c#L1455
https://github.com/opencomputeproject/SAI/blob/f69394f590362b1b02852ec80f00f9a27a02ed97/meta/saisanitycheck.c#L1489